.github/workflows/kyverno-policy.yaml

```
@@ -43,8 +43,8 @@ jobs:
         
       - name: Wait for clusterpolicies
         run: |
-          kubectl wait --for=condition=Ready clusterpolicy/argo-workflow
-          kubectl wait --for=condition=Ready clusterpolicy/visit-member
+          kubectl wait --for=condition=Ready clusterpolicy/argo-workflow --timeout=90s
+          kubectl wait --for=condition=Ready clusterpolicy/visit-member --timeout=90s
```

Can make second one immediate

```suggestion
          kubectl wait --for=condition=Ready clusterpolicy/argo-workflow --timeout=90s
          kubectl wait --for=condition=Ready clusterpolicy/visit-member --timeout=0s
```